nuxt/pages/about.vue

```diff
@@ -98,7 +115,6 @@
 </template>
 
 <script lang="ts">
-/* eslint-disable global-require */
 import { Component, Vue } from 'vue-property-decorator';
```

Comment:
I forgot, one small change here:
```suggestion
import { Component, Vue } from 'nuxt-property-decorator';
```